modules/prebidmanagerAnalyticsAdapter.js

```diff
@@ -0,0 +1,193 @@
+import {ajaxBuilder} from '../src/ajax';
+import adapter from '../src/AnalyticsAdapter';
+import adapterManager from '../src/adapterManager';
+
+/**
+ * prebidmanagerAnalyticsAdapter.js - analytics adapter for prebidmanager
+ */
+const DEFAULT_EVENT_URL = 'https://endpoint.prebidmanager.com/endpoint'
+const analyticsType = 'endpoint';
+const analyticsName = 'Prebid Manager Analytics: ';
+
+var utils = require('src/utils');
+var CONSTANTS = require('src/constants.json');
+let ajax = ajaxBuilder(0);
+
+function generateUUID() { // Public Domain/MIT
+  var d = new Date().getTime();
+  if (typeof performance !== 'undefined' && typeof performance.now === 'function') {
+    d += performance.now(); // use high-precision timer if available
+  }
+  return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, function (c) {
+    var r = (d + Math.random() * 16) % 16 | 0;
+    d = Math.floor(d / 16);
+    return (c === 'x' ? r : (r & 0x3 | 0x8)).toString(16);
+  });
+}
+
+var _VERSION = 1;
+var initOptions = null;
+var _pageViewId = generateUUID();
+var _startAuction = 0;
+var _bidRequestTimeout = 0;
+var pmAnalyticsEnabled = false;
+
+var w = window;
+var d = document;
+var e = d.documentElement;
+var g = d.getElementsByTagName('body')[0];
+var x = w.innerWidth || e.clientWidth || g.clientWidth;
+var y = w.innerHeight || e.clientHeight || g.clientHeight;
+
+var _pageView = {
+  eventType: 'pageView',
+  userAgent: window.navigator.userAgent,
+  timestamp: Date.now(),
+  timezoneOffset: new Date().getTimezoneOffset(),
+  language: window.navigator.language,
+  vendor: window.navigator.vendor,
+  screenWidth: x,
+  screenHeight: y
+};
+
+var _eventQueue = [
+  _pageView
+];
+
+let prebidmanagerAnalytics = Object.assign(adapter({url: DEFAULT_EVENT_URL, analyticsType}), {
+  track({eventType, args}) {
+    handleEvent(eventType, args);
+  }
+});
+
+prebidmanagerAnalytics.originEnableAnalytics = prebidmanagerAnalytics.enableAnalytics;
+prebidmanagerAnalytics.enableAnalytics = function (config) {
+  initOptions = config.options;
+  initOptions.url = initOptions.url || DEFAULT_EVENT_URL;
```

Comment:
If a user enables the `prebidManager` Analytics without providing at least an empty `options` param, then this will cause an error and Prebid will not work:

Example:

```javascript
pbjs.enableAnalytics({
                'provider': 'prebidmanager'
            });
```

![image](https://user-images.githubusercontent.com/13972794/56610318-f9791180-65c3-11e9-9ac4-ffd3d23d06f1.png)


Please add a check to make sure `config.options` exists before assigning it to `initOptions`

Perhaps:
```javascript
initOptions = config.options || {};
```
?

Reply:
done. Thank you.